doc/source/groupby.rst

```diff
@@ -1014,6 +1014,22 @@ Regroup columns of a DataFrame according to their sum, and sum the aggregated on
    df
    df.groupby(df.sum(), axis=1).sum()
 
+Groupby by Indexer to 'resample' data.
+
+Resampling produces new hypothetical samples(resamples) from already existing observed data or from a data generating mechanism which resemble the underlying population.
```

Comment:
"data generating mechanism which resemble the underlying population" seems a bit difficult explanation.